### PR TITLE
Lg/nns draft

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
This is a **very preliminary** sketch just from the Slack message (no overview/how-to/workflow, just bare bones of the methods):
David Ribeiro Alves:spiral_calendar_pad:  4 hours ago
@team-nns working on producing a doc today that summarizes the MVP integration points that coinbase needs. Planning on including and describing these methods, Comments welcome:
create_neuron(locked_icpt, ledger_account, dissolve_period) - Authenticated with the “cold” key - Transfers ‘locked_icpt’ from a ledger account controlled by the caller and creates a new neuron with the set dissolve_period. The neuron is not dissolving.
add_hotkey(neuron_id) - Authenticated with the “cold” key - Adds a hot key to the list of authorized keys of the neuron.
remove_hotkey(neuron_id) - Authenticated with the “cold” key - Removes a hot key to the list of authorized keys of the neuron.
get_neuron_info(neuron_id) - Authenticated with cold key or hot keys - Returns information about the neuron.
start_dissolving(neuron_id)  - Authenticated with the “cold” key - Starts the dissolve timer.
stop_dissolving(neuron_id) - Authenticated with the “cold” key - Stops the dissolve timer.
dissolve_neuron(neuron_id) - Authenticated with the “cold” key - If the timer is 0 withdraws all the staked_icpt and transfers them to the callers ledger account.

### CODE REVIEW

File: modules/integration/pages/nns-quick-start.adoc [hunk 1]
```diff
@@ -0,0 +1,126 @@
+= Integrating with the Internet Computer governance system
+:proglang: Motoko
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:2]: lang is Rust, not sure whether thast matters

File: modules/integration/pages/nns-quick-start.adoc [hunk 2]
```diff
@@ -0,0 +1,126 @@
+= Integrating with the Internet Computer governance system
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:1]:
```suggestion
= Integrating with the Internet Computer governance system - The Network Nervous System (NNS)
```

File: modules/integration/pages/nns-quick-start.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+:proglang: Motoko
+:platform: Internet Computer platform
+:IC: Internet Computer
+:company-id: DFINITY
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+
+[[nns-intro]]
+This _{doctitle}_ guide introduces the Internet Computer Network Nervous System (NNS). 
+The Internet Computer Network Nervous System (NNS) is a decentralized, algorithmic governance system that manages network operations for (ICP) subnets and stakeholders.
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:10]:
```suggestion
The Internet Computer Network Nervous System (NNS) is a decentralized, algorithmic governance system that manages many aspects of the the Internet Computer (operations, economics, etc.) on behalf of its stakeholders.
```

File: modules/integration/pages/nns-quick-start.adoc [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+The document includes an overview of the NNS and its key operations and provides a summary of the application programming interface (API) methods that support staking operations for ICP token holders.
+
+_{doctitle}_ is intended as a high-level overview for organizations and developers who need to understand the terminology and operational management flow for participants in the Internet Computer network who want to have a stake and voting rights by locking some number of ICP tokens for a given period of time.
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:14]: nit this sentence seems a bit too long

File: modules/integration/pages/nns-quick-start.adoc [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+In reviewing this document, keep in mind that additional details about specific components or interfaces might be available in subsequent documents to supplement the overview provided in this document.
+In addition, this overview focuses on how to integrate with the Internet Computer using the link:https://www.rosetta-api.org/docs/welcome.html[Rosetta API].
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:17]: not sure this is true, think we might be skipping rosetta. for now let's keep it out

File: modules/integration/pages/nns-quick-start.adoc [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+== Basic terminology to get you started
+
+The Internet Computer is primarily a distributed and decentralized platform for running software. 
+When developers write applications that run on the Internet Computer, they deploy their programs in the form of a conceptual computational unit called a **canister**. 
+A canister is similar to a “smart contract” in that it consists of both program code and state running, replicated, on a block chain network with security and liveness guarantees.
+
+
+
+== NNS overview
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:29]: think you can get a lot of these definitions from Dom's doc. I would suggest we stay true to that document as much as possible.

File: modules/integration/pages/nns-quick-start.adoc [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+=== Neurons
+
+=== Proposals
+
+=== Voting
+
+== Integration with the NNS using the Rosetta API
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:37]: remove rosetta API stuff

File: modules/integration/pages/nns-quick-start.adoc [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+=== Basic integration workflow
+
+The following steps summarize the basic operational workflow for .... 
+
+
+=== How to ... 
+
+This section summarizes the steps for how to 
+
+=== Requirements and limitations
```
Comment (dralves) [inline modules/integration/pages/nns-quick-start.adoc:55]: Here I'd mention that this document only covers the minimal API for neuron custody and staking. Participating in governance, including voting, is not in scope.

**Review by dralves — APPROVED** [review]

**Comment by dralves** [comment]
Approved, just some minor suggestions.

File: modules/integration/pages/nns-quick-start.adoc (hunk 8 above)
Comment (lsgunnlsgunn) [reply modules/integration/pages/nns-quick-start.adoc:55]: I'll add this maybe to the intro.

**Pull request merged**